docs/examples/Irrigation.ipynb

```diff
@@ -2,37 +2,47 @@
  "cells": [
```

Comment:
<p>Hard to see that lines 1,4,5 are the concept due to lots of plotting code. Can the cell be split on line 6 and the comment be promoted to Markdown?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Okay</p>